db/scripts/find_cured.py

```diff
@@ -20,7 +21,7 @@ def find_cured_NC(state):
 
   print("Connected to db")
 
-  cursor = mydb.cursor()
+  cursor = mydb.cursor(MySQLdb.cursors.DictCursor)
```

Comment:
Not sure why this was originally changed. It was giving issues: https://stackoverflow.com/questions/35359969/typeerror-tuple-indices-must-be-integers-not-str

Reply:
Oh I removed it since we weren't accessing the rows using their col names

Reply:
I see, it ended up having issues without it so I added it back